packages/@react-spectrum/s2/stories/SelectBoxGroup.stories.tsx

```diff
@@ -72,141 +72,49 @@ const meta: Meta<typeof SelectBoxGroup> = {
       control: 'select',
       options: ['vertical', 'horizontal']
     },
-    numColumns: {
-      control: {type: 'number', min: 1, max: 4}
-    },
-    gutterWidth: {
-      control: 'select',
-      options: ['compact', 'default', 'spacious']
-    },
-    showCheckbox: {
-      control: 'boolean'
-    }
+    selectedKeys: {control: false, table: {disable: true}},
+    defaultSelectedKeys: {control: false, table: {disable: true}}
   },
   args: {
     selectionMode: 'single',
     orientation: 'vertical',
-    numColumns: 2,
-    gutterWidth: 'default',
-    isDisabled: false,
-    showCheckbox: false
+    isDisabled: false
   }
 };
 
 export default meta;
 type Story = StoryObj<typeof meta>;
 
 export const Default: Story = {
-  render: (args) => (
-    <SelectBoxGroup {...args} onSelectionChange={action('onSelectionChange')}>
-      <SelectBox value="aws">
-        <Server />
-        <Text slot="label">Amazon Web Services</Text>
-        <Text slot="description">Reliable cloud infrastructure</Text>
-      </SelectBox>
-      <SelectBox value="azure">
-        <AlertNotice />
-        <Text slot="label">Microsoft Azure</Text>
-      </SelectBox>
-      <SelectBox value="gcp">
-        <PaperAirplane />
-        <Text slot="label">Google Cloud Platform</Text>
-      </SelectBox>
-      <SelectBox value="ibm">
-        <StarFilled1 />
-        <Text slot="label">IBM Cloud</Text>
-        <Text slot="description">Hybrid cloud solutions</Text>
-      </SelectBox>
-    </SelectBoxGroup>
-  )
-};
-
-export const MultipleSelection: Story = {
-  args: {
-    selectionMode: 'multiple',
-    defaultSelectedKeys: new Set(['aws', 'gcp']),
-    numColumns: 3,
-    gutterWidth: 'default'
-  },
-  render: (args) => (
-    <div style={{maxWidth: 800}}>
-      <p style={{marginBottom: 16, fontSize: 14, color: '#666'}}>
-        Focus any item and use arrow keys for grid navigation:
-      </p>
-      <SelectBoxGroup {...args} onSelectionChange={action('onSelectionChange')}>
+  render: (args) => {
+    const {selectionMode, orientation, isDisabled} = args as any;
+    return (
+      <SelectBoxGroup selectionMode={selectionMode} orientation={orientation} isDisabled={isDisabled} UNSAFE_style={{gridTemplateColumns: 'repeat(2, 1fr)'}}>
```

Comment:
you can just spread args onto here once you have the types fixed

we shouldn't use UNSAFE_style in our documentation, I'd just get rid of it, and if you need to constrain the area so it doesn't go across the entire screen, just add a wrapping div with an appropriate width. Check other storybooks for examples